bindings/python/.evergreen/test.sh

```diff
@@ -42,14 +50,22 @@ else
              "/opt/python/3.6/bin/python3"
              "/opt/python/pypy/bin/pypy"
              "/opt/python/pypy3.6/bin/pypy3")
+    export CSFLE_PATH="../csfle/lib/mongo_csfle_v1.so"
+    /opt/mongodbtoolchain/v3/bin/python3 drivers-evergreen-tools/.evergreen/mongodl.py --component \
+      csfle --version latest --out ../csfle/ --target rhel70
 fi
 
+
 for PYTHON_BINARY in "${PYTHONS[@]}"; do
     echo "Running test with python: $PYTHON_BINARY"
     $PYTHON_BINARY -c 'import sys; print(sys.version)'
     createvirtualenv $PYTHON_BINARY .venv
     python -m pip install --prefer-binary -r test-requirements.txt
     python setup.py test
+    DYLD_FALLBACK_LIBRARY_PATH=../csfle/lib/:$DYLD_FALLBACK_LIBRARY_PATH \
```

Comment:
Please add a comment or better a echo line that says something like "Running tests with csfle on the system's library search path"